### PR TITLE
Chris/style

### PR DESCRIPTION
**Pull request by ChrisTienYenWalker**
Modified Contact and Footer

Changes:

Footer: Added team information and logo and template for socials

Contact page: Added location, Google Maps embedded website and email contact

Proof of functionality (screenshot, logs, etc):
I showed you it, I haven't made changes since I showed it to you
image
image

Additional Notes
I didn't make any comments but I named everything well

Checklist
Your code builds clean without any errors or warnings
Code has been formatted (clang-format, cmake-format, black, prettier, rustfmt, etc)
Where possible, unit tests have been added
Code is well-commented & understood

### CODE REVIEW

File: src/assets/main.css [hunk 1]
```diff
@@ -3,7 +3,7 @@
 :root {
     --gryphon-red: #C20430;
     --gryphon-yellow: #FFC72A;
-    --gryphon-black: #000000;
+    --gryphon-black: #110f0fee;
```
Comment (Nomalah) [inline src/assets/main.css]: This doesn't follow the university style guide, if you want to use a different color, make a different variable

File: src/components/NavFooter.vue [hunk 2]
```diff
@@ -1,7 +1,90 @@
...
+                        <img />
+                    </li>
+                    <li>
+                        <img />
+                    </li>
+                </ul>
+            </div>
+
+            <div class="teamLogo">
+                <NuxtLink to="/"><img src="../../public/media/GryphonBanner.svg" /></NuxtLink>
```
Comment (Nomalah) [inline src/components/NavFooter.vue]:
```suggestion
                <RouterLink to="/"><img src="/media/GryphonBanner.svg" /></RouterLink>
```
not using nuxt anymore

File: src/views/ContactView.vue [hunk 3]
```diff
@@ -1,3 +1,91 @@
...
+  --c:  var(--gryphon-yellow); /* the color*/
+  height:10vh;
+  width: 20vw;
+  box-shadow: 0 0 0 .1em inset var(--c); 
+  --_g: linear-gradient(var(--c) 0 0) no-repeat;
+  background: 
+    var(--_g) calc(var(--_p,0%) - 100%) 0%,
+    var(--_g) calc(200% - var(--_p,0%)) 0%,
+    var(--_g) calc(var(--_p,0%) - 100%) 100%,
+    var(--_g) calc(200% - var(--_p,0%)) 100%;
```
Comment (Nomalah) [inline src/views/ContactView.vue]: What the buns is this?

**Review by Nomalah — CHANGES_REQUESTED** [review]
Can you remove the .nuxt directory and the app.vue file?
Add some screenshots as we don't have autodeploy for feature branches.
Run `npm run lint`

**Comment by ChrisTienYenWalker** [comment]
Your router redirect is not working

**Review by Nomalah — APPROVED** [review]

**Pull request merged**